vue/components/ui/molecules/form-input/form-input.vue

```diff
@@ -99,6 +133,19 @@ export const FormInput = {
         success: {
             type: String,
             default: null
+        },
+        alignment: {

```

Comment:
I understand why you chose this name, but since we're using this for left/center/right in other components let's change it to `variant`, which is the name we're using for "style variations" kind of stuff.